lib/Model.php

```diff
@@ -415,7 +415,7 @@ public function __set($name, $value)
 		foreach (static::$delegate as &$item)
 		{
 			if (($delegated_name = $this->is_delegated($name,$item)))
-				return $this->$item['to']->$delegated_name = $value;
+				return $this->{$item['to']}->$delegated_name = $value;
```

Comment:
[I mentioned this in the upstream PR](https://github.com/jpfuentes2/php-activerecord/pull/553#discussion_r88141490), but we should always be using the curly-brace wrappers whenever we nest references like this. I'd suggest changing this to:

```php
return $this->{$item['to']}->{$delegated_name} = $value;
```